syft/src/main/java/org/openmined/syft/datamodels/JobDataModel.kt

```diff
@@ -0,0 +1,42 @@
+package org.openmined.syft.datamodels
+
+import java.security.MessageDigest
+
+/**
+ * A uniquer identifier class for the job
+ * @property modelName The name of the model used in the job while querying PyGrid
+ * @property version The model version in PyGrid
+ */
+class JobDataModel(
```

Comment:
This class should have a name describing its purpose. What about `JobId`?

Reply:
Also, consider making it a `data class`

Reply:
> This class should have a name describing its purpose. What about `JobId`?

There is an inner JobId class inside SyftJob, I guess I should remove It and make this one right?

Reply:
That would be good.